alternator/extract_from_attrs.hh

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2023-present ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: AGPL-3.0-or-later
```

Comment:
This needs updating now to point to the new license (here and in regular_column_transformation).